dgp/proto/annotations.proto

```diff
@@ -213,6 +222,68 @@ message Polygon2DAnnotation{
   map<string, string> attributes = 3;
 }
 
+// 3D point.
+message KeyPoint3D {
+  // (x, y, z) point (in 3D carthesian coordinates).
+  double x = 1;
+  double y = 2;
+  double z = 3;
+}
+
+// 3D point annotation.
+message KeyPoint3DAnnotation {
+  // Class identifier (should be in [0, num_classes - 1])
+  uint32 class_id = 1;
+
+  // 3D point.
+  KeyPoint3D point = 2;
+
+  // An associative map stores arbitrary attributes, where the key is attribute name
+  // and the value is attribute value. Both key_type and value_type are string.
+  map<string, string> attributes = 3;
```

Comment:
Could you please share where we can find documentation of keys and value semantics to expect here?